connectivity/http/http.c

```diff
@@ -256,9 +251,9 @@ static int ta_http_process_request(ta_http_t *const http, iota_client_service_t
     return process_options_request(out);
   }
 
-  if (api_path_matcher(url, "/mam/[A-Z9]{81}[/]?") == SC_OK) {
```

Comment:
No generic matcher required?

Reply:
This matcher was served for `recv_mam_message()`. It was a GET method API, but now it became a POST method API. Therefore, we don't need a  generic matcher here.